frontend/src/App/State/AppSectionState.ts

```diff
@@ -53,6 +53,7 @@ export interface AppSectionItemSchemaState<T> {
 export interface AppSectionItemState<T> {
   isFetching: boolean;
   isPopulated: boolean;
+  isSaving: boolean;
```

Comment:
`isSaving` is not present for all sections, this should be added to a more specific type instead.

Reply:
Reverted